cli/cluster/collectorfactory.go

```diff
@@ -26,17 +27,17 @@ type collectorsFactory struct{}
 
 // NewCollectorFactory creates a new CollectorFactory.
 func NewCollectorFactory() CollectorFactory {
```

Comment:
```suggestion
func NewDataCollectorFactory() integrity.DataCollectorFactory {
```
+
a decorator method would be more expected that we not have:

1. `DataCollectorFactory` in `integrity` it should be decorated to `CollectorFactory`.
2. `CollectorFactory` in `cluster`.

That confusing and asymmetrical.

Reply:
Done, now `NewDataCollectorFactory` returns an `integrity.DataCollectorFactory` as well